core/libs/models/src/api.rs

```diff
@@ -409,5 +410,66 @@ impl Request for GetSubscriptionInfoRequest {
     const ROUTE: &'static str = "/get-subscription-info";
 }
 
+#[derive(Serialize, Deserialize, Debug, PartialEq, Clone)]
+pub struct AdminDeleteAccountRequest {
+    pub username: String,
+}
+
+#[derive(Serialize, Deserialize, Debug, PartialEq, Clone)]
+pub enum AdminDeleteAccountError {
+    Unauthorized,
+    UsernameNotFound,
+}
+
+impl Request for AdminDeleteAccountRequest {
+    type Response = ();
+    type Error = AdminDeleteAccountError;
+    const METHOD: Method = Method::DELETE;
+    const ROUTE: &'static str = "/admin-delete-account";
+}
+
+#[derive(Serialize, Deserialize, Debug, PartialEq, Eq, Hash, Clone)]
+pub enum FeatureFlag {
+    NewAccounts,
+}
+
+#[derive(Serialize, Deserialize, Debug, PartialEq, Clone)]
+pub struct ToggleFeatureFlagRequest {
+    pub feature: FeatureFlag,
+    pub enable: bool,
+}
+
+#[derive(Serialize, Deserialize, Debug, PartialEq, Clone)]
+pub enum ToggleFeatureFlagError {
+    Unauthorized,
+}
```

Comment:
You can re-use errors if they're the same, just call this `AuthError` or something like that